pkg/client/clientset/versioned/typed/scheduling/v1beta1/scheduling_client.go

```diff
@@ -46,9 +46,7 @@ func (c *SchedulingV1beta1Client) Queues() QueueInterface {
 // NewForConfig creates a new SchedulingV1beta1Client for the given config.
 func NewForConfig(c *rest.Config) (*SchedulingV1beta1Client, error) {
 	config := *c
-	if err := setConfigDefaults(&config); err != nil {
-		return nil, err
-	}
+	setConfigDefaults(&config)
```

Comment:
those are generated code, so please do not modify them.

Reply:
done